_data/navigation.yml

```diff
@@ -91,6 +91,8 @@ setup:
       - title: Configure Artifact Support
         collapsed: true
         children:
+          - title: Overview
```

Comment:
Added this to have something to link to when mentioning "configure your artifacts"